Sources/AppcuesKit/Data/Models/ScreenshotUpload.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ScreenshotUpload.swift
+//  AppcuesKit
+//
+//  Created by James Ellis on 2/15/23.
+//  Copyright © 2023 Appcues. All rights reserved.
+//
+
+import Foundation
+
+// response model for the customer API pre-upload-screenshot endpoint
+internal struct ScreenshotUpload: Decodable {
```

Comment:
I think I like the idea of grouping the all screen capture stuff _somewhere_. Maybe even a top level directory alongside Data/Presentation/etc? Not sure the right home, but I don't like the idea of having it scattered throughout.

Reply:
Yeah, good call. I'll look into structure clean up there.

Reply:
moved them all into `/Presentation/Debugger/ScreenCapturing` for now, to be together at least. This is in the part 3 PR on top of this in commit https://github.com/appcues/appcues-ios-sdk/pull/343/commits/b851302eb4d4a820fd512a91e633df49288ded4f -- will squash these down.